docs/source/modin/supported/window_supported.rst

```diff
@@ -20,7 +20,8 @@ Rolling window functions
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``apply``                   | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``corr``                    | N                               |                                                    |
+| ``corr``                    | P                               | ``N`` for non-integer ``window``, ``axis = 1``,    |
+|                             |                                 |  ``pairwise = True`` or ``min_periods != window``  |
```

Comment:
I think you also have to add ``N`` for `other = None`. Also, your doctest uses `pairwise = None`.

Reply:
Right, adding other=None here. And I think ``N`` for ``pairwise = True`` should hold true since pairwise is a bool value with default None in the pandas [docs](https://pandas.pydata.org/pandas-docs/version/2.1/reference/api/pandas.core.window.rolling.Rolling.corr.html)?